tests/sync-mpsc.rs

```diff
@@ -361,25 +388,6 @@ fn oneshot_multi_thread_send_close_stress() {
     }
 }
 
-#[test]
-fn oneshot_multi_thread_recv_close_stress() {
-    for _ in 0..stress_factor() {
-        let (tx, rx) = channel::<i32>();
-        thread::spawn(move || {
-            let res = thread::spawn(move || {
-                rx.recv().unwrap();
-            })
-            .join();
-            assert!(res.is_err());
-        });
-        let _t = thread::spawn(move || {
-            thread::spawn(move || {
-                drop(tx);
-            });
-        });
-    }
-}
```

Comment:
Could you use something like `#[cfg_attr(not(feature = "mustang_noignore"), ignore)]` on these tests instead of grouping them  to comment them out, so that we keep the diff with the upstream version of this file simple?

Reply:
i group them together because i thought they were affecting each other, since unlike all the other tests, these fail spurriously and i only observed the failure when all of them were running.

i tried this again by running each one in a loop, and found that only `oneshot_multi_thread_recv_close_stress`
was causing a segfault.

because an abort doesn't flush stdout, its not always clear at what point the abort happens, with the other tests i could retry them once alone to see if the segfault happens again, but not with this one.

Reply:
i would also rather stick to using `cargo test`s ability to run ignored test with `cargo test ... -- --ignored` rather than introduce a new feature for this.